packages/core-data/src/resolvers.js

```diff
@@ -136,6 +136,9 @@ export function* getEntityRecord( kind, name, key = '', query ) {
 
 		const record = yield apiFetch( { path } );
 		yield receiveEntityRecords( kind, name, record, query );
+	} catch ( error ) {
+		// We need a way to handle and access REST API errors in state
+		// Until then, catching the error ensures the resolver is marked as resolved.
```

Comment:
I don't understand how having an empty `catch` changes something.. 🤔

Reply:
haha :) an empty catch allows the resolver to "finish" and `@wordpress/data` will make the resolver as "resolved".

Reply:
I'm sorry but I still don't get it... :). Can you elaborate a bit or send me a link of the code where this change affects something? Because even if there is no `catch` clause it will still go to `finally` if is rejected.

Reply:
It will go to finally but won't finish the execution of the resolver, basically it will kill the execution of the resolver after the  finally and throw the error.

Reply:
See the console here https://codesandbox.io/s/trusting-ramanujan-4w3ys?file=/src/index.js:24-36 and then remove the "catch" block and see the result.

Reply:
Okay, I get what you're saying now and it made me curious because in this specific function there was no other code to be executed afterwards, but I haven't checked in depth how wp.data works and obviously makes some other code not to run. 

Thank you for your time to make me understand it Riad 👍